tests/functional/n/name_styles.txt

```diff
@@ -1,16 +1,17 @@
-invalid-name:50:CorrectClassName.__DunDER_IS_not_free_for_all__:"Method name ""__DunDER_IS_not_free_for_all__"" doesn't conform to snake_case naming style":INFERENCE
-invalid-name:80::"Class name ""BAD_NAME_FOR_CLASS"" doesn't conform to PascalCase naming style"
-invalid-name:81::"Class name ""NEXT_BAD_NAME_FOR_CLASS"" doesn't conform to PascalCase naming style"
-invalid-name:88::"Class name ""NOT_CORRECT"" doesn't conform to PascalCase naming style"
-invalid-name:94:test_globals:"Constant name ""AlsoCorrect"" doesn't conform to UPPER_CASE naming style"
-invalid-name:107:FooClass.PROPERTY_NAME:"Attribute name ""PROPERTY_NAME"" doesn't conform to snake_case naming style":INFERENCE
-invalid-name:112:FooClass.ABSTRACT_PROPERTY_NAME:"Attribute name ""ABSTRACT_PROPERTY_NAME"" doesn't conform to snake_case naming style":INFERENCE
-invalid-name:117:FooClass.PROPERTY_NAME_SETTER:"Attribute name ""PROPERTY_NAME_SETTER"" doesn't conform to snake_case naming style":INFERENCE
+invalid-name:9::"Constant name ""bad_const_name"" doesn't conform to UPPER_CASE naming style"
+invalid-name:12:BADFUNCTION_name:"Function name ""BADFUNCTION_name"" doesn't conform to snake_case naming style"
+invalid-name:14:BADFUNCTION_name:"Variable name ""BAD_LOCAL_VAR"" doesn't conform to snake_case naming style"
+invalid-name:18:func_bad_argname:"Argument name ""NOT_GOOD"" doesn't conform to snake_case naming style"
+invalid-name:28:bad_class_name:"Class name ""bad_class_name"" doesn't conform to PascalCase naming style"
+invalid-name:39:CorrectClassName.__init__:"Attribute name ""_Bad_AtTR_name"" doesn't conform to snake_case naming style"
+invalid-name:40:CorrectClassName.__init__:"Attribute name ""Bad_PUBLIC_name"" doesn't conform to snake_case naming style"
+invalid-name:45:CorrectClassName.BadMethodName:"Method name ""BadMethodName"" doesn't conform to snake_case naming style":INFERENCE
+invalid-name:51:CorrectClassName.__DunDER_IS_not_free_for_all__:"Method name ""__DunDER_IS_not_free_for_all__"" doesn't conform to snake_case naming style":INFERENCE
+invalid-name:81::"Class name ""BAD_NAME_FOR_CLASS"" doesn't conform to PascalCase naming style"
+invalid-name:82::"Class name ""NEXT_BAD_NAME_FOR_CLASS"" doesn't conform to PascalCase naming style"
+invalid-name:89::"Class name ""NOT_CORRECT"" doesn't conform to PascalCase naming style"
+invalid-name:95:test_globals:"Constant name ""AlsoCorrect"" doesn't conform to UPPER_CASE naming style"
+invalid-name:108:FooClass.PROPERTY_NAME:"Attribute name ""PROPERTY_NAME"" doesn't conform to snake_case naming style":INFERENCE
+invalid-name:113:FooClass.ABSTRACT_PROPERTY_NAME:"Attribute name ""ABSTRACT_PROPERTY_NAME"" doesn't conform to snake_case naming style":INFERENCE
+invalid-name:118:FooClass.PROPERTY_NAME_SETTER:"Attribute name ""PROPERTY_NAME_SETTER"" doesn't conform to snake_case naming style":INFERENCE
+invalid-name:149:FooEnum:"Constant name ""bad_enum_name"" doesn't conform to UPPER_CASE naming style"
```

Comment:
I wonder if we should have a more specialized message such as "Enum name ... doesn't conform to UPPER_CASE naming style" but maybe the definition of a specialized check for that doesn't worth it.
Another idea would be too change the message with: "Constant/Enum name 
"..." doesn't conform to UPPER_CASE naming style"
What do you think about it?

Reply:
> I wonder if we should have a more specialized message such as "Enum name ... doesn't conform to UPPER_CASE naming style"

I agree.
But I couldn't implement it because I didn't have a deep understanding of the pylint code.
I need some ideas or hints.

> Another idea with: "Constant/Enum name "..." doesn't conform to UPPER_CASE naming style"

In [the discussion](https://bugs.python.org/issue40721), they interpretation that Enums are Constants.
"Constant/Enum" is redundant, I think.

Reply:
@Hee-San you are right it is redundant. Let's try with a common checker for Constant and Enum!